applications/nrf_desktop/configuration/nrf52840dk_nrf52840/sysbuild.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+
+SB_CONFIG_SECURE_BOOT_APPCORE=y
+SB_CONFIG_SECURE_BOOT_SIGNING_KEY_FILE="b0_private.pem"
```

Comment:
Do we still need to set the `CONFIG_SB_SIGNING_KEY_FILE` in application configuration? Similar for other Kconfig options for B0 bootloader.

Reply:
It is separate ticket that was not addressed yet here -> [NCSDK-27890](https://nordicsemi.atlassian.net/browse/NCSDK-27890)